presets.js

```diff
@@ -49,11 +48,9 @@ export function getPresets() {
 				],
 			}
 		})
-	}
 
-	if (this.switch.switchStatsPort) {
-		this.switch.switchStatsPort.forEach((port) => {
-			presets[`poe_${port.portId}`] = {
+		self.port_stats.all().forEach((port) => {
+			presets[`link_${port.portId}`] = {
```

Comment:
Little bug here – these preset handles were clobbering the others, so there were only one set of presets (POE ones) – this makes `Link Status` work in the UI